modules/serverpod_auth/serverpod_auth_server/lib/src/endpoints/firebase_endpoint.dart

```diff
@@ -37,29 +37,24 @@ class FirebaseEndpoint extends Endpoint {
       session.log('Verified idToken', level: LogLevel.debug);
       var claims = token.claims;
 
-      // Verify that we at a minimum got the email address.
-      if (claims.email == null) {
-        return AuthenticationResponse(
-          success: false,
-          failReason: AuthenticationFailReason.invalidCredentials,
-        );
-      }
-
-      var email = claims.email!.toLowerCase();
+      var email = claims.email?.toLowerCase();
       var userIdentifier = token.claims.subject;
-      var userName = token.claims.nickname ?? email.split('@')[0];
       var fullName = token.claims.name;
+      var userName = token.claims.nickname ?? email?.split('@')[0];
```

Comment:
This will throw if we have en empty array right? Might as well guard against it now that we modify this code.

```suggestion
      var userName = token.claims.nickname ?? email?.split('@').firstOrNul();
```